### PR TITLE
Add django-mptt-admin

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@conda-forge/staged-recipes: Could you please review and merge if everything looks okay.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Can you please review and merge if everything looks okay.

File: recipes/django-mptt-admin/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (CJ-Wright) [inline recipes/django-mptt-admin/meta.yaml]: Can you change this over to pip? https://conda-forge.org/docs/meta.html#use-pip

Comment (sannykr) [reply recipes/django-mptt-admin/meta.yaml]: Thanks for the review. Please let me know if you want more changes

**Comment by CJ-Wright** [comment]
Seems reasonable.

**Pull request merged**